Makefile.am

```diff
@@ -96,3 +96,8 @@ include $(top_srcdir)/makefiles/test-cases.mak
 
 include $(top_srcdir)/makefiles/help.mak
 
+if HAVE_RST2MAN
+%.1.in: %.1.in.rst
```

Comment:
Thank you for comment.


The rule is for helping a developer who writes the man page.
I expect such person may install both rst2man and GNU make.

`.in` file is recorded to our git repository. The `.in` file is for people who doesn't modify the file.
In such case the rule will never used; `configure` just generates the man page from the `.in` file.